### ISSUE TITLE
[Feature request] Support Arabic and Devanagari

### ISSUE THREAD

**Issue by stone-zeng** [be5invis/Iosevka#320]
Please consider to add Arabic and Devanagari support (in the future).

One of the (open source) mono-spaced Arabic font is  [Kawkab Mono](https://makkuk.com/kawkab-mono/), you may refer to it for glyph design. Courier New supports Arabic and Devanagari etc, but the design is apparently different.

**Comment by be5invis** [comment]
Typesetting complex script in a monospace manner is still not a well-studied area. It can somehow analogue to justification.
I'd like to mark it far future...

**Comment by habibalamin** [comment]
@stone-zeng you may be interested in Pragmata Pro, which is similar to Iosevka and also has Arabic support.

**Comment by clsn** [comment]
If you eventually go down this route, consider tossing in Hebrew, which is *way* less complicated than Arabic or Devanagari (and that's even if you go nuts getting all the vowels and stuff looking good.)

**Comment by simurgh9** [comment]
I love Kawkab! I also love Iosevka and would love to have it be my all rounder font but unfortunately it does not support Arabic/Urdu/Persian. If I can help answer any questions as someone who knows how to read and write those scripts. I'd be happy to.

**Comment by yuvadm** [comment]
@clsn Opened #690 to discuss potential Hebrew glyph support

**Comment by Naheel-Azawy** [comment]
I agree with @stone-zeng and @simurgh9. I believe Kwakab would look perfect with Iosevka. Here's a classical showcase:
![image](https://user-images.githubusercontent.com/5764614/94722115-813d8080-035f-11eb-8ad2-9fa43966e231.png)

**Comment by Pepsiman-12** [comment]
In Iosevka Mono and Iosevka Term, would you also make the lam alif ligature 2 characters wide while the regular Iosevka shows it as 1 character wide?

**Comment by simurgh9** [comment]
Does it even support Arabic type languages now?

Checked, it does not. This is the only thing keeping me from using this font right now.

**Comment by saeedtahmtan** [comment]
# Suggestion
there are opensource fonts i just want to mention because they are very good for this purpose 

## Estedad: [github](https://github.com/aminabedi68/Estedad)
it not monospace but have great appearance 

## Vazir-code-font [github](https://github.com/rastikerdar/vazir-code-font)
it  is monospace but archived 

> those are the best options that anyone can find for Persian and Arabic fonts the Estedad font is best option in my opinion.

**Comment by t4rik** [comment]
Monospaced font support Arabic (Open Sourced)
https://github.com/microsoft/cascadia-code
